packages/ariakit-store/readme.md

```
@@ -4,14 +4,234 @@
 
 ```ts
 import { createStore } from "@ariakit/store";
 ```
 
 This package is ESM-only and exposes a single public entrypoint.
+
+<!-- ariakit-docs:start -->
+
+## API reference
+
+#### `createStore`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)